packages/block-editor/src/components/global-styles/style.scss

```diff
@@ -83,13 +83,18 @@
 	}
 
 	.block-editor-global-styles-background-panel__image-tools-panel-item {
+		padding: 0;
 		flex-grow: 1;
 		border: 0;
 		.components-dropdown {
```

Comment:
I know it was not introduced by this PR, but — as also mentioned about `.components-button` — `.components-*` classnames re not meant to be used outside of the components package. We can use custom classnames instead.

Reply:
Should I make the change in the PR only, or a separate PR would be good for it?

Reply:
Happy for it to be a separate, follow-up PR.

The main changes that would be needed:
- understand why we need to set a `36px` height on `.components-dropdown`. Ideally, we should use the default button height. If not, let's understand why, and at most use the `renderToggle` prop to pass differently sized button;
- avoid overriding color and focus styles of `.components-button`, and only use the component props to tweak those aspects (mainly the `variant` prop)
- passing a custom `className` to the components instances, and then using that custom classname in the styles file;